src/composables/useNetwork.ts

```diff
@@ -1,18 +1,30 @@
-import { Network } from '@balancer-labs/sdk';
 import { computed, ref } from 'vue';
 
 import config from '@/lib/config';
 import { configService } from '@/services/config/config.service';
+import { Network } from '@balancer-labs/sdk';
 
 /**
  * STATE
  */
-const DEFAULT_NETWORK_ID =
-  process.env.VUE_APP_NETWORK != null
-    ? (Number(process.env.VUE_APP_NETWORK) as Network)
-    : Network.MAINNET;
+const windowAvailable = typeof window !== 'undefined';
+const localStorageNetworkId: Network | undefined =
+  windowAvailable && localStorage.getItem('networkId')
+    ? (Number(localStorage.getItem('networkId')) as Network)
+    : undefined;
+
+const NETWORK_ID =
+  localStorageNetworkId ||
+  Number(process.env.VUE_APP_NETWORK) ||
```

Comment:
VUE_APP_NETWORK should be deprecated with these changes. It shouldn't be required anymore.

Reply:
I think we still need it for unit tests... Not sure if possible to run some of the tests which require kovan without it.